### PR TITLE
5111 - Session key resign-in prompt only shows option that was used

### PR DESCRIPTION
**Pull request by lzach83**
## Link to Issue
Closes: #5111 

## Description of Changes
- Added a find to compare the selected address with the available wallets addresses 
- Only show the selected addresses' auth option

## "How We Fixed It"

- Added a .find method to compare state to the selected wallet address and grab that auth method

### CODE REVIEW

**Comment by masvelio** [comment]
lgtm 👍

**Comment by masvelio** [comment]
lgtm 👍

File: packages/commonwealth/client/scripts/views/modals/SessionRevalidationModal/SessionRevalidationModal.tsx [hunk 1]
```diff
@@ -19,6 +19,8 @@ import TerraWalletConnectWebWalletController from 'controllers/app/webWallets/te
 import { X } from '@phosphor-icons/react';
 import { openConfirmation } from 'views/modals/confirmation_modal';
 import { setActiveAccount } from 'controllers/app/login';
+import IWebWallet from 'client/scripts/models/IWebWallet';
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/modals/SessionRevalidationModal/SessionRevalidationModal.tsx:22]: Please do not use `client/scripts/` prefixed imports, per our [Code Requirements](https://github.com/hicommonwealth/commonwealth/blob/master/wiki/Code-Requirements.md)

File: packages/commonwealth/client/scripts/views/modals/SessionRevalidationModal/SessionRevalidationModal.tsx [hunk 2]
```diff
@@ -77,6 +79,15 @@ const SessionRevalidationModal = ({
   const chainbase = app.chain?.meta?.base;
   const wallets = WebWalletController.Instance.availableWallets(chainbase);
 
+  const findSelectedWallet = () => {
+    const selectedWallet = app.user.addresses.find(
+      (w) => w.address === walletAddress
+    );
+    return selectedWallet
+      ? [wallets.find((wallet) => wallet.name === selectedWallet.walletId)]
+      : [selectedWallet];
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/modals/SessionRevalidationModal/SessionRevalidationModal.tsx:88]: What's this branch of the conditional doing? Returning `[null]`?

**Review by jnaviask — CHANGES_REQUESTED** [review]
Couple Qs/feedback.

**Comment by lzach83** [comment]
Updated the function to no longer return an odd `[null] ` and updated the import to follow the coding standard more closely.

**Comment by jnaviask** [comment]
@lzach83 rather than using a massive relative import, you can use the absolute import with `client/scripts/` omitted: `models/IWebWallet`. Any folder _within_ `client/scripts/` is valid as an import prefix.

**Comment by jnaviask** [comment]
@lzach83 your modification to the return type of the `selectedWallet` conditional will actually cause the `hideSocialLogins={findSelectedWallet().length > 0}` line to cause an error, as it now returns a null/undefined value rather than an array. Please fix.

**Comment by jnaviask** [comment]
@lzach83 one last comment -- can we add the return type to the function rather than using an explicit `as` cast?

**Comment by lzach83** [comment]
@jnaviask Updated, should be good now

**Pull request merged**